vagrant/provisioning/roles/foia/templates/arkcase-portal-server-2020.16.yaml

```diff
@@ -0,0 +1,40 @@
+logging.level.web: DEBUG
+
+# server.port: 8080
+# server.servlet.context-path: /${portal.applicationName}
+
+# Property used for enable/disable Spring security. For authenticated portal comment this property.  
+#spring.autoconfigure.exclude: org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration,org.springframework.boot.actuate.autoconfigure.security.servlet.ManagementWebSecurityAutoConfiguration
+
+
+portal.externalConfiguration: true
+portal.host: {{ external_host }}
+portal.port:
+portal.uiApplicationUrl: "https://{{ external_host }}/foia"
+portal.uiApplicationHost: "https://{{ external_host }}"
+portal.id: {{ 9999999999999999999999 | random | to_uuid }}
+
+portal.authenticatedMode: false
```

Comment:
I can't help but notice that `portal.authenticatedMode` is set to `false` and I don't see anything that would change it to `true`.  Am I missing anything?

Reply:
Or is this configuration somehow changed to the ArkCase admin page can manage `portal.authenticatedMode`?

Reply:
Yes. It can be changed now from admin page. Although, I would add this as a variable for different servers. And there is a DSO ticket for that.